docs/static/breaking-changes.asciidoc

```diff
@@ -29,7 +30,155 @@ and have the persistent queue enabled, we strongly recommend that you drain or
+
+*Beats Input*
+
+* Removed obsolete `congestion_threshold` option
+* Removed obsolete `target_field_for_codec` option
+* Changed default value of `add_hostname` to false
+
+NOTE: In Beats 7.0.0, the fields exported by Beats _to_ the Logstash Beats Input
+conform to the {ecs-ref}/index.html[Elastic Common Schema (ECS)]. Many of the
+exported fields have been renamed, so you may need to modify your pipeline
+configurations to access them at their new locations prior to upgrading your
+Beats. See {beats-ref}/breaking-changes-7.0.html[Beats Breaking changes in 7.0]
+for the full list of changed names.
+
+*HTTP Input*
+
+* Removed obsolete `ssl_certificate_verify` option
+
+*HTTP Poller Input*
+
+* Removed obsolete `interval` option
+* Removed obsolete `ssl_certificate_verify` option
+
+*Tcp Input*
+
+* Removed obsolete `data_timeout` option
+* Removed obsolete `ssl_cacert` option
+
+[float]
+===== Output Plugins
+
+*Elasticsearch Output*
+
+* Remove support for parent/child (still support join data type) since we don't support multiple document types any more
+* Removed obsolete `flush_size` option
+* Removed obsolete `idle_flush_time` option
+
+*HTTP Output*
+
+* Removed obsolete `ssl_certificate_verify` option
+
+*Kafka Output*
+
+* Removed obsolete `block_on_buffer_full` option
+* Removed obsolete `ssl` option
+* Removed obsolete `timeout_ms` option
+
+*Redis Output*
+
+* Removed obsolete `queue` option
+* Removed obsolete `name` option
+
+*Sqs Output*
+
+* Removed obsolete `batch` option
+* Removed obsolete `batch_timeout` option
+
+*Tcp Output*
+
+* Removed obsolete `message_format` option
+
 [float]
+===== Beats input
+
+Beats fields have been renamed to map to Elastic Common Schema (ECS).
+
+Starting with 7.0, the fields exported by {beats} conform to the
+{ecs-ref}/index.html[Elastic Common Schema (ECS)]. Many of the exported fields
+have been renamed. See {beats-ref}/breaking-changes-7.0.html[Beats breaking
+changes in 7.0] for the full list of changed names.
+
+See the *{beats} Platform Reference* for more information on
+{beats-ref}/upgrading-6-to-7.html#enable-ecs-compatibility[Beats and ECS].
+
+TODO:  What action does user need to take?  
+
+[float]
+===== Elasticsearch output
+
+ILM on by default
+
+TODO:  What action does user need to take?  
+
+ - The Field Reference parser, which is used to interpret references to fields in your pipelines and plugins, was made to be more strict and will now reject inputs that are either ambiguous or illegal; since 6.4, Logstash has emitted warnings when encountering input that is ambiguous, and allowed an early opt-in of strict-mode parsing either by providing the command-line flag `--field-reference-parser STRICT` or by adding `config.field_reference.parser: STRICT` to `logstash.yml`.
+
+
+[float]
+===== List of plugins bundled with Logstash
+
+These plugins were removed from the 7.0 default bundle based on usage data. 
+You can still install these plugins manually.
+
+* removed-plugin-1
```

Comment:
only the graphite output stopped being supported but it's still in the package

Reply:
Any notation required around that? 
I'll remove this section entirely.

Reply:
I think we can remove this section, commercial support is something only for our customers, which can be checked in our support matrix page